jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -1123,57 +1123,37 @@ code_blockt java_string_library_preprocesst::make_assign_function_from_call(
 ///   primitive value contained in the given object.
 optionalt<symbol_exprt>
 java_string_library_preprocesst::get_primitive_value_of_object(
-  const exprt &object,
-  irep_idt type_name,
-  const source_locationt &loc,
-  const irep_idt &function_id,
-  symbol_table_baset &symbol_table,
-  code_blockt &code)
-{
+    const exprt &object, irep_idt type_name, const source_locationt &loc,
+    const irep_idt &function_id, symbol_table_baset &symbol_table,
+    code_blockt &code) {
   optionalt<struct_tag_typet> object_type;
 
   typet value_type;
-  if(type_name==ID_boolean)
-  {
-    value_type=java_boolean_type();
+  if (type_name == ID_boolean) {
+    value_type = java_boolean_type();
```

Comment:
Still a lot of reformatting in here?!

Reply:
should be better now

Reply:
Hmm, then the changes are actually being reverted in a later commit - this still looks very much broken.

Reply:
It's "Remove code that is now unreachable" that removes the mis-formatted code.